xtask/README.md

```
@@ -41,21 +40,19 @@ cargo xtask test-e2e [OPTIONS]
 *   `--scenario <SCENARIO>`: Sets the network conditions for the test.
     *   `basic-connectivity` (Default): No adverse network conditions are applied.
     *   `high-latency`: Simulates a high-latency network (100ms RTT).
-    *   `packet-loss`: Simulates a network with 10% packet loss.
+    *   `packet-loss`: Simulates a network with 5% packet loss.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)